core/divesite.c

```diff
@@ -10,6 +10,7 @@
 
 struct dive_site_table dive_site_table;
 
+// TODO: keep table sorted by UUID and do a binary search?
```

Comment:
Since we don't match by uuid, I'm not sure how that would be reasonable. What am I missing?

Reply:
When parsing and when creating new UUIDs, we *do* search by UUID. In this case a binary search might help. It's just that
1) The comment precedes the wrong function.
2) With the later changes the table *does* stay sorted by UUID. Therefore the comment is not correct.

I will check that out.

Reply:
Moved it to the correct function.